examples/pytorch-federated-variational-autoencoder/pyproject.toml

```diff
@@ -1,15 +1,39 @@
-[tool.poetry]
-name = "pytorch_federated_variational_autoencoder"
-version = "0.1.0"
-description = "Federated Variational Autoencoder Example"
-authors = ["The Flower Authors <hello@flower.ai>"]
-
-[tool.poetry.dependencies]
-python = ">=3.8,<3.11"
-flwr = ">=1.0,<2.0"
-torch = "1.13.1"
-torchvision = "0.14.1"
-
 [build-system]
-requires = ["poetry-core>=1.4.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "fedvaeexample"
+version = "1.0.0"
+description = "Federated Variational Autoencoder Example with PyTorch and Flower"
+license = "Apache-2.0"
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+]
```

Comment:
```suggestion
```

Reply:
we've only been keeping this for those examples that had external contributors. I think in this case it's not needed